doc/dev.md

```diff
@@ -116,115 +116,4 @@ residing in its own top-level directory:
 * `grease-cli`: This defines a command-line application on top of the library
   code in `grease`.
 
-## Test suite
```

Comment:
Is it worth including a forward reference to `doc/tests.md` here?

Reply:
Added under "building", as the normal sequence is build -> test!